# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1269459941 14400
#      Wed Mar 24 15:45:41 2010 -0400
# Node ID e85fa73ac3a03a3830a5d924bc8e8d760ca6c354
# Parent  f123146f1820c0cfb73aa6eb7915b7a0101e27f6
src/DLD-FUNCTIONS/config-module.awk: emit additions to EXTRA_DIST

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,12 @@
+2010-03-24  John W. Eaton  <jwe@octave.org>
+
+	* DLD-FUNCTIONS/config-module.awk: Emit additions to EXTRA_DIST.
+
 2010-03-24  John W. Eaton  <jwe@octave.org>
 
 	* Makefile.am ($(srcdir)/DLD-FUNCTIONS/module.mk):
 	Remove doubled // from file name in rule.
 
 2010-03-24  David Grundberg  <davidg@cs.umu.se>
 
 	* oct-parse.yy: Include gnulib stdlib.h/stdio.h instead of C++
diff --git a/src/DLD-FUNCTIONS/config-module.awk b/src/DLD-FUNCTIONS/config-module.awk
--- a/src/DLD-FUNCTIONS/config-module.awk
+++ b/src/DLD-FUNCTIONS/config-module.awk
@@ -1,24 +1,31 @@
 BEGIN {
   print "## DO NOT EDIT -- generated from module-files by config-module.awk";
+  print ""
+  print "EXTRA_DIST += \\"
+  print "  DLD-FUNCTIONS/config-module.sh \\"
+  print "  DLD-FUNCTIONS/config-module.awk \\"
+  print "  DLD-FUNCTIONS/module-files"
+  print ""
   nfiles = 0;
 } {
   files[++nfiles] = $1;
 } END {
   sep = " \\\n";
   print "DLD_FUNCTIONS_LIBS = \\";
   for (i = 1; i <= nfiles; i++) {
     basename = files[i];
     sub (/\.cc$/, "", basename);
     if (i == nfiles)
       sep = "\n";
     printf ("  DLD-FUNCTIONS/%s.la%s", basename, sep);
   }
   print "octlib_LTLIBRARIES += $(DLD_FUNCTIONS_LIBS)";
+  print ""
   print "if AMCOND_ENABLE_DYNAMIC_LINKING";
   for (i = 1; i <= nfiles; i++) {
     basename = files[i];
     sub (/\.cc$/, "", basename);
     printf ("DLD-FUNCTIONS/%s.oct: DLD-FUNCTIONS/%s.la\n", basename, basename);
     print "\trm -f $@";
     print "\tla=`echo $< | $(SED) 's,DLD-FUNCTIONS/,,'` && \\";
     print "\t  of=`echo $@ | $(SED) 's,DLD-FUNCTIONS/,,'` && \\";
